Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -195,47 +195,52 @@ internal sealed class PythonLanguageClient : ILanguageClient, ILanguageClientCus
                     await JoinableTaskContext.Factory.SwitchToMainThreadAsync();
                 }
 
-                object settings = null;
-                if (item.section == "python") {
-                    Debug.Assert(_analysisOptions != null);
+                Debug.Assert(_analysisOptions != null);
 
-                    var extraPaths = UserSettings.GetStringSetting(
-                        PythonConstants.ExtraPathsSetting, null, Site, PythonWorkspaceContextProvider.Workspace, out _)?.Split(';')
-                        ?? _analysisOptions.ExtraPaths;
+                var extraPaths = UserSettings.GetStringSetting(
+                    PythonConstants.ExtraPathsSetting, null, Site, PythonWorkspaceContextProvider.Workspace, out _)?.Split(';')
+                    ?? _analysisOptions.ExtraPaths;
 
-                    var stubPath = UserSettings.GetStringSetting(
-                        PythonConstants.StubPathSetting, null, Site, PythonWorkspaceContextProvider.Workspace, out _)
-                        ?? _analysisOptions.StubPath;
+                // Add search paths to extraPaths for pylance to look through
+                var searchPaths = context.SearchPaths.ToArray();
```

Comment:
do search paths have any alias strings in them VS related ie. {ProjectFolder}

Reply:
Yes they do. Good catch. I need to expand them.